main.go

```diff
@@ -325,5 +336,42 @@ func makeRequest(t transport.Transport, request *transport.Request) (*transport.
 	ctx, cancel := tchannel.NewContext(request.Timeout)
 	defer cancel()
 
+	if tracer := t.Tracer(); tracer != nil {
+		span := tracer.StartSpan(request.Method)
+		for k, v := range request.Baggage {
+			span = span.SetBaggageItem(k, v)
+		}
+		ctx = opentracing.ContextWithSpan(ctx, span)
+	}
+
 	return t.Call(ctx, request)
 }
+
+func makeInitialRequest(out output, transport transport.Transport, serializer encoding.Serializer, req *transport.Request) {
```

Comment:
I am not sure where this new code is coming from. There might be rebase magic here.